lib/utils.pm

```diff
@@ -2118,8 +2118,7 @@ sub assert_secureboot_status {
 }
 
 sub susefirewall2_to_firewalld {
-    my $timeout = 360;
-    $timeout = 1200 if is_aarch64;
+    my $timeout = 1200;
```

Comment:
Your VR show 360s needed on power, what's timeout for other platform? such as 390?

Reply:
Not sure, just run many cases it will fail. I think we should use platform at here.